remo/annotation_utils.py

```diff
@@ -0,0 +1,29 @@
+import csv
+
+# file_name,class_name,coordinates
+# ILSVRC2012_val_00000003.JPEG,N01751748, 10 20 30 40
+from remo.domain import Annotation
+
+
+def parse_plain_csv_object_detection(file_path):
+    """
+    Args
+        file_path: path to annotations
+
+    :return: annotations
```

Comment:
@vmanilo can you please add a docstring?
also, what's annotations?

Reply:
added example